Resources/Prototypes/GameRules/roundstart.yml

```
@@ -209,9 +205,8 @@
       components:
       - type: Revolutionary
       - type: HeadRevolutionary
-      mindComponents:
-      - type: RevolutionaryRole
-        prototype: HeadRev
+      mindRoles:
+      - MindRoleHeadRevolutionary
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)